### DISCUSSION TITLE
How to fix graph size.

### DISCUSSION THREAD

**Discussion by piturski** [NightscoutFoundation/xDrip#2606] [Q&A]
SInce yesterday, my graph size is lower. How can I fix it?
![Screenshot_20230107_122506_xDrip+](https://user-images.githubusercontent.com/26544445/211147880-7c429355-fe86-4e6c-ac43-b8b02b9c6dea.jpg)

**Comment by Navid200** [comment]
Would you please post a screenshot of the classic status page?
https://navid200.github.io/xDrip/docs/xDrip-Version

You can blackout the transmitter ID if you like.

**Reply by piturski** [reply]
I have OOPAlgorith2, Libre 2 and Raw data without calibration.

**Comment by Navid200** [comment]
@jwoglom What determines the position of the stop/start flags on the graph?  
What could cause them to move so high on the graph?  Please see the attached image.
![211147880-7c429355-fe86-4e6c-ac43-b8b02b9c6dea](https://user-images.githubusercontent.com/51497406/211154308-66dc295b-db02-4f8c-90e7-0001242e04d0.jpg)

**Reply by piturski** [reply]
That was start of Libre2 sensor by NFC scan, without calibration(raw data from OPP2)

**Reply by Navid200** [reply]
Has it been 24 hours since then where the flags are shown on your screen?

**Reply by piturski** [reply]
No, currently 21 hours since flag event.

**Reply by Navid200** [reply]
When it has been 24 hours, it will clear out and the vertical axis will return to normal.  If it does not, please let me know by posting here in this same thread.

**Reply by jwoglom** [reply]
@Navid200 i'm not sure but based on looking through my own xdrip history events I suspect it is somewhat related to the estimated value of the current BG given that it always appears around the CGM trend at that time.

**Reply by Navid200** [reply]
Makes sense.  Thanks.
I suspect in this case the raw values were very high and that's why the flags were placed at such high values.

**Reply by piturski** [reply]
This is event log
![Screenshot_20230107_231423_xDrip+](https://user-images.githubusercontent.com/26544445/211172241-4fa1c54d-f948-47d7-90a9-2cb3ac28a74f.jpg)
![Uploading Screenshot_20230107_231438_xDrip+.jpg…]()

**Reply by Navid200** [reply]
@piturski You mentioned that the problem was resolved, right?

**Reply by piturski** [reply]
Yes :)

**Reply by Navid200** [reply]
Awesome.   Thanks

**Comment by piturski** [comment]
![Screenshot_20230107_174113_xDrip+](https://user-images.githubusercontent.com/26544445/211161509-56923cf3-f7db-4df9-be7e-eff826776aa5.jpg)

**Comment by piturski** [comment]
Currently size of a graph is correct. You can close it.
![Screenshot_20230107_204410_xDrip+](https://user-images.githubusercontent.com/26544445/211167987-3fac3f4d-3089-4e51-b2df-bf763d338440.jpg)

**Reply by Navid200** [reply]
There is nothing to close.
This is a forum.  You can post and ask any questions you may have.  
Everyone is welcome to ask questions and everyone is welcome to participate and try to help.

**Comment by yaiza88** [comment]
I have a similar problem, I use xdrip follow dex, when connecting the glucometer via bluetooth the graph disappears, how can I solve it?
![photo_2023-02-17_17-15-38](https://user-images.githubusercontent.com/68350271/219720880-0adfe1a8-b918-4aa3-a9d4-43965937624f.jpg)

**Reply by Navid200** [reply]
To post a screenshot, please scale the image so that it still shows without covering the entire screen when opened.  Then, click on the bottom border when editing your post and you will see the option to choose an image.  Chose the image file and it will be added to your post.  There is no reason to add a link to an image.

**Reply by yaiza88** [reply]
I tried to upload the photo, I don't know if I did it right, can you tell me? sorry my english is translated by google

**Reply by Navid200** [reply]
Yes, I can see the image you uploaded.  
Please look at the image I am uploading now.   It is a cropped out section from your image.
What happened where I have marked?  You have a single point at a very high value and that's why the y axis range has changed so that that point can be shown.
![219720880-0adfe1a8-b918-4aa3-a9d4-43965937624f](https://user-images.githubusercontent.com/51497406/219722666-44e9326e-be32-4ee9-9293-06489ada928f.jpg)

**Reply by yaiza88** [reply]
![glucometro](https://user-images.githubusercontent.com/68350271/219728013-120197ea-26ad-4a4e-85f8-18de1046a208.jpg)
That point is when I measure with a glucometer, in that case the glucometer gave me an error and it comes out like this

**Reply by yaiza88** [reply]
It occurred to me to try to delete that error from the glucometer and the problem has disappeared. Is there an option to change glucometer error notes?

**Reply by Navid200** [reply]
Do you mean every time you use your Bluetooth blood glucose meter, this happens?

**Reply by yaiza88** [reply]
When doing a blood test, it is registered in the graph, but this glucometer error had never happened to me, causing the error in the graph. When reading the measurement label, it registered the error number 2046, I think that was registered as a glucose test real

**Reply by Navid200** [reply]
I think you are saying no, this does not happen every time.
If that is what you are saying, then, I don't think we have a problem.  If the blood glucose meter causes an error once a year, you can delete the error and your graph will never be distorted.

**Reply by yaiza88** [reply]
Yes. It had never happened to me, neither the error in the graph, nor the glucometer error, hence I looked for a post with the same or similar problem because I did not know what it could be, I was scared after 24 hours like that without discovering what it could be failure. Thanks to the fact that you commented on the red dot so high, otherwise I would not have realized that this could be my problem. Thank you!!!
